javascript/chronometer.js

```diff
@@ -1,34 +1,58 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime=0,
+    this.intervalId= null
   }
 
-  start(callback) {
-    // ... your code goes here
-  }
+  start(callback) 
+  {
+    if(!callback)
+    {
+    this.intervalId= setInterval(()=> this.currentTime++,1000)     
+    }
 
-  getMinutes() {
-    // ... your code goes here
+    else {
+    this.intervalId= setInterval(()=> this.currentTime++,1000)
+    callback() 
+    }
+     
+  }
+  getMinutes() 
+  {
+    return (Math.floor(this.currentTime/60))
   }
 
-  getSeconds() {
-    // ... your code goes here
+  getSeconds() 
+  {
+    return (this.currentTime % 60)
   }
 
-  computeTwoDigitNumber(value) {
-    // ... your code goes here
+  computeTwoDigitNumber(value) 
+  {
+    if (value<10)
+    {
+      return "0" + value 
```

Comment:
Me parece qu este tenia que ser un string si lo interpolamos con backticks podemos devolverlo como un string
```suggestion
      return `0${value}`
```